### DISCUSSION TITLE
Not able to Open the Lab

### DISCUSSION THREAD

**Discussion by united10** [ibm-skills-network/author-workbench-knowledgebase#52] [Help]
https://author.skills.network/courses/607/labs/3082/lti_redirect

Not able to open this Link for IBM ETL and Data Pipelines with Shell, Airflow and Kafka course from coursera 

![image](https://github.com/ibm-skills-network/author-workbench-knowledgebase/assets/42668685/8be07256-9ca6-4391-aa35-fec1cb539fb8)

**Comment by michaelroudnitski** [comment]
Hi @united10, this can happen when the lab launch is interrupted. Can you please try launching the lab again from Coursera? If the problem persists, please contact your course instructors to let them know there is a problem with the lab
